docs/notes/notes.rst

```diff
@@ -5,115 +5,25 @@
 Information about the release lifecycle can be found
 `here <https://github.com/eProsima/Fast-DDS/blob/master/RELEASE_SUPPORT.md>`_.
 
-Version 2.13.3
-==============
+.. include:: previous_versions/v2.13.3.rst
 
-This patch release includes the following **improvements**:
-
-1. Enabling multiple interfaces through whitelist in TCP servers
-2. Set LARGE_DATA Participants logic with the same listening ports
-3. Check History QoS inconsistencies
-
-This patch release includes the following **fixes**:
+.. seealso::
 
-1. Prevent index overflow and correctly assert the end iterator in DataSharing
-2. Fix the shared memory cleaning script
-3. Fix TCP reconnection after open logical port failure
-4. Fix data race on PDP
-5. Fix doxygen docs warnings. Prepare for compiling with Doxygen 1.10.0
-6. Add missing TypeLookup listeners
-7. Restore Blackbox tests names
-8. Add macOS Github CI
-9. Set 2.11.x as EOL
+    For further information about the corresponding versions of other products related to this Fast DDS version, please
+    refer to the :ref:`dependencies_compatibilities_product_compatibility` section.
 
-.. note::
+Supported versions
```

Comment:
I think I'd actually put both supported versions and EOL versions at the same level as the latest in the TOC. I'd be something like:
1. Dependencies and compatibility
2. Version 2.13.3 (latest)
3. Supported versions
4. Previous end-of-life versions